management/backup.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python3
+# -*- coding: utf-8 -*-
```

Comment:
You mean it beeing gray? Thats probably due to hashtag beeing a comment in python by default